grail/drand.py

```
@@ -1,147 +1,400 @@
+- Uniform schema for real & mock beacons
+"""
+from __future__ import annotations
+
+import os
+import json
+import random
+import logging
+import requests
+from threading import Lock
+from typing import Any, Dict, Optional, Tuple
+from typing import Any, Dict, Optional, Tuple
```

Ruff (0.12.2)

Redefinition of unused `Any` from line 38

Remove definition: `Any`

(F811)